### PR TITLE
Disable + remove org admin tool fields

### PR DESCRIPTION
**Pull request by rluodev**
This closes issue #4698.

### CODE REVIEW

File: app/models/event.rb [hunk 1]
```diff
@@ -536,6 +536,8 @@ def fee_balance_v2_cents
   def plan_name
     if unapproved?
       "pending approval"
+    elsif high_school_hackathon? && sponsorship_fee == 0
+      "full fiscal sponsorship (fee waived)"
```
Comment (sampoder) [inline app/models/event.rb]:
```suggestion
      "pending approval"
    elsif hack_club_hq?
      "hack club project"
    elsif salary?
      "salary account"
    elsif high_school_hackathon? && sponsorship_fee == 0
      "full fiscal sponsorship (fee waived)"
```

how about adding a couple more of these? i think this suggestions covers them all

**Review by sarthaktexas — COMMENTED** [review]
Approved! Just make sure to commit Sam's stuff and then I'll change mine to an approval as well. I haven't tried this locally so I can formally approve it.

File: app/views/events/_form.html.erb [hunk 2]
```diff
@@ -124,7 +124,7 @@
       </div>
     <% end %>
 
-    <% if @event.category == "hackathon" || @event.category == "event" %>
+    <% if @event.category == "hackathon" || @event.category == "event" || @event.category == "high school hackathon" %>
```
Comment (garyhtou) [inline app/views/events/_form.html.erb]:
```suggestion
    <% if @event.category.in? ["hackathon", "event", "high school hackathon"] %>
```

**Comment by rluodev** [comment]
@sarthaktexas @sampoder @garyhtou merged Sam’s stuff!

File: app/models/event.rb [hunk 3]
```diff
@@ -536,6 +536,12 @@ def fee_balance_v2_cents
   def plan_name
     if unapproved?
       "pending approval"
+    elsif hack_club_hq?
+      "hack club project"
+    elsif salary?
+      "salary account"
+    elsif high_school_hackathon? && sponsorship_fee == 0
```
Comment (garyhtou) [inline app/models/event.rb]: Does this only apply to high school hackathons? Or should it be any Event that has a zero sponsorship fee

Comment (sarthaktexas) [inline app/models/event.rb]: why isn't this line? @garyhtou also mentioned this: https://github.com/hackclub/hcb/pull/4701#discussion_r1424478275
```suggestion
    elsif sponsorship_fee == 0
```

**Review by sampoder — APPROVED** [review]

**Review by sarthaktexas — APPROVED** [review]
I haven't tested this locally because I can't. It looks to work though, and I also trust Sam.

**Comment by rluodev** [comment]
@garyhtou this should be good to go

File: app/models/event.rb [hunk 4]
```diff
@@ -536,6 +536,12 @@ def fee_balance_v2_cents
   def plan_name
     if unapproved?
       "pending approval"
+    elsif hack_club_hq?
+      "hack club project"
```
Comment (garyhtou) [inline app/models/event.rb]:
```suggestion
      "hack club affiliated project"
```

File: app/views/events/_form.html.erb [hunk 5]
```diff
@@ -243,7 +243,7 @@
 
   <% admin_tools do %>
     <div class="field" id="admin_settings">
-      <%= form.label :name, "Name (if changed, also change in G Suite)" %>
+      <%= form.label :name, "Name" %>
```
Comment (garyhtou) [inline app/views/events/_form.html.erb]:
```suggestion
      <%= form.label :name %>
```

**Review by garyhtou — APPROVED** [review]

**Pull request merged**